src/services/gamestate.service.js

```diff
@@ -0,0 +1,98 @@
+const games = {
```

Comment:
Why do we have an object with dummy data here? Maybe we should create a data type similar to the card and deck models.

Reply:
I think we should not use a data model before we have implemented all features regarding gamestates. Maybe something will change. The dummy data is used to give an idea of the structure can be removed if not needed.

Reply:
Ok, then let's remove the dummy data